core/modules/language/language.admin.inc

```diff
@@ -155,20 +155,31 @@ function language_admin_add_form($form, &$form_state) {
 }
 
 /**
- * Editing screen for a particular language.
+ * This function was replaced by language_admin_config_form() but is preserved
+ * for backwards-compatibility.
+ *
+ * @deprecated since 1.13.3
+ */
+function language_admin_edit_form($form, &$form_state, $language) {
+  watchdog_deprecated_function('language', __FUNCTION__);
+  return language_admin_config_form($form, $form_state, $language);
+}
+
+/**
+ * Configuration screen for a particular language.
  *
  * @param $langcode
  *   Language code of the language to edit.
  */
-function language_admin_edit_form($form, &$form_state, $language) {
+function language_admin_config_form($form, &$form_state, $language) {
   _language_admin_common_controls($form, $language);
   $form['actions'] = array('#type' => 'actions');
   $form['actions']['submit'] = array(
     '#type' => 'submit',
     '#value' => t('Save language')
   );
-  $form['#submit'][] = 'language_admin_edit_form_submit';
-  $form['#validate'][] = 'language_admin_edit_form_validate';
+  $form['#submit'][] = 'language_admin_config_form_submit';
+  $form['#validate'][] = 'language_admin_config_form_validate';
```

Comment:
Should we add wrapper/deprecated functions for the renamed ones?

Reply:
Those don't get altered by contrib.